golangci-lint/.golangci.yml

```diff
@@ -76,7 +83,8 @@ linters:
     - forbidigo
     - forcetypeassert
     - funlen
-    - gci
+    # gci would need to be customized for each project in order to be useful
+    #- gci
```

Comment:
If we do not customize it in each project it is not useful. For instance:

```go
import (
	"flag"
	"fmt"
	"io"
	"os"

	"github.com/scaleway/scaleway-sdk-go/logger"
	log "github.com/sirupsen/logrus"

	"github.com/newrelic-forks/newrelic-prometheus/configurator"
)
```

would need the configuration:

```yaml
linters-settings:
  gci:
    sections:
      - standard # Captures all standard packages if they do not match another section.
      - default # Contains all imports that could not be matched to another section type.
      - prefix(github.com/newrelic-forks/newrelic-prometheus) # Group for the local project
```

Reply:
Seems very redundant with `goimports` too. LGTM.

Reply:
why commenting out and not deleting it?

Reply:
I normally prefer to delete instead of comment, but I think in this case it would save time to future us if we ever decide to enable it again.